djangoproject/settings/common.py

```diff
@@ -217,8 +217,16 @@
 
 TIME_ZONE = "America/Chicago"
 
+# Internationalization settings
+
 USE_I18N = True
 
+LOCALE_PATHS = [
+    # List apps first so their translations take precedence.
+    # This list should match LOCALE_DIRS in update-translations.sh.
+    str(BASE_DIR.joinpath(path))
+    for path in ("dashboard/locale/", "docs/locale/", "locale/")
```

Comment:
Django already loads `locale` directories from installed apps: https://docs.djangoproject.com/en/5.2/topics/i18n/translation/#how-django-discovers-translations

So I think this can be simplified to:
```suggestion
    BASE_DIR.joinpath("locale"),
```